lib/QubitCsvImport.class.php

```diff
@@ -35,8 +35,20 @@ class QubitCsvImport
   public $indexDuringImport = false;
   public $doCsvTransform = false;
 
-  public function import($csvFile, $type = null)
+  public function import($csvFile, $type = null, $csvOrigFileName = null)
   {
+    if (null === $csvOrigFileName)
```

Comment:
Yes, I considered doing what you're describing, but opted for readability as there are a bunch of comments there to explain what's going on.